src/main/java/de/tum/cit/aet/artemis/communication/service/notifications/GroupNotificationScheduleService.java

```diff
@@ -59,12 +80,22 @@ public void checkAndCreateAppropriateNotificationsWhenUpdatingExercise(Exercise
      */
     private void checkAndCreateExerciseReleasedNotificationsWhenUpdatingExercise(Exercise exerciseBeforeUpdate, Exercise exerciseAfterUpdate) {
         final ZonedDateTime initialReleaseDate = exerciseBeforeUpdate.getReleaseDate();
+        checkAndCreateExerciseReleasedNotificationsWhenUpdatingExerciseWithDate(initialReleaseDate, exerciseAfterUpdate);
+    }
+
+    /**
+     * Auxiliary method that checks and creates exercise-released notifications when updating an exercise
+     *
+     * @param originalReleaseDate is the initial release date before it gets updated
+     * @param exerciseAfterUpdate is the updated exercise (needed to check potential difference in release date)
+     */
```

Comment:
I do not understand what `(needed to check potential difference in release date)` is supposed to tell me as a callee; that comment confuses me.

I do also think that `     * Auxiliary method that checks and creates exercise-released notifications when updating an exercise` is contained in the method name and does not need to be repeated